advisories/github-reviewed/2021/09/GHSA-mc22-5q92-8v85/GHSA-mc22-5q92-8v85.json

```diff
@@ -18,7 +18,7 @@
     {
       "package": {
         "ecosystem": "crates.io",
-        "name": "Tremor"
+        "name": "tremor-runtime"
```

Comment:
There is no crate called `tremor-runtime`. The only crate that is affected by it, is https://crates.io/crates/tremor-script

```suggestion
        "name": "tremor-script"
```

Reply:
Thank you for the clarification!